### PR TITLE
user list 1190

### LINKED ISSUE #1190: Implement Account - User list

### PR DESCRIPTION
**Pull request by na3d**
Closes #1190

### CODE REVIEW

**Comment by na3d** [comment]
screen shot 2017-03-10 at 4 13 10 pm

**Comment by na3d** [comment]
screen shot 2017-03-10 at 4 18 55 pm

File: scss/components/cards/SecondaryCard.scss [hunk 1]
```diff
@@ -16,4 +19,10 @@
         margin-right: 15px;
         display: none;
     }
+
+    .user-icon {
+        line-height: 50px;
```
Comment (mparke) [inline scss/components/cards/SecondaryCard.scss]: :( don't use `line-height`, use `padding` if needed

File: src/users/layouts/IndexPage.js [hunk 2]
```diff
@@ -13,16 +33,51 @@ export class IndexPage extends Component {
...
+    return (
+      <div className="PrimaryPage container">
+        <header className="PrimaryPage-header">
+          <div className="PrimaryPage-headerRow clearfix">
+            <h1 className="float-sm-left">Users</h1>
+          </div>
+        </header>
+        <div className="PrimaryPage-body">
+          <div className="row">
+            {Object.values(users.users).map(user =>
```
Comment (mparke) [inline src/users/layouts/IndexPage.js]: we should begin to use `collection.ids` where possible, to retain order if the api wrapper has sorted something.

so in this case,

```
users.ids.map((id) => {
  // access by id
  const user = users.users[id];
  
});

Comment (mparke) [inline src/users/layouts/IndexPage.js]: may need to break into a function, ( or module + define as function ) so it's easy to reference the user obj

Comment (na3d) [inline src/users/layouts/IndexPage.js]: > `Issue created`

File: src/users/layouts/IndexPage.js [hunk 3]
```diff
@@ -13,16 +33,51 @@ export class IndexPage extends Component {
...
+              <div className="col-lg-6" key={user.username}>
+                <SecondaryCard
```
Comment (mparke) [inline src/users/layouts/IndexPage.js]: can we rename this to something meaningful?

**Pull request merged**